exercises/sum-of-multiples/description.md

```diff
@@ -1,4 +1,5 @@
-Given a number, find the sum of all the multiples of particular numbers up to but not including that number.
+Given a number, find the sum of all the multiples of particular numbers up to
```

Comment:
This is a good point but I'm not going to change this as part of this PR as it is not related to the initial duplication.